docs/admin-guide.md

```diff
@@ -0,0 +1,42 @@
+# ChainPatrol
+
+### About this app
+
+The [ChainPatrol](https://chainpatrol.io/) mini-app gives your community members
+access to security tools to check URLs for scams and report malicious activity
+on your Discord server. Use the `/chainpatrol <sub-command>` commands to empower
+your community to protect themselves from phishing attacks and other scams.
+
+### Getting Started
+
+To get started, install the ChainPatrol mini-app from the
+[Collab.Land Marketplace](https://cc.collab.land/).
+
+Your community members can then start using `/chainpatrol check` to start
+checking URLs whenever they are unsure of a message that has been sent in your
+Discord server. If they find a scam, they can use the `/chainpatrol report`
+command to report the scam to ChainPatrol.
+
+### Configuration
+
+Without configuration, reports will be sent to the ChainPatrol team for review.
+If you would like to be able to review reports and keep note of which community
+members are helping protect your server, you can reach out to the ChainPatrol
+team to get a custom admin dashboard associated to your server.
+
+---
+
+<center>
+
+[Get access to admin dashboard](https://chainpatrol.io/beta)
```

Comment:
is this required or optional?

Reply:
Optional, added in brackets to the subheading